libs/state/docs/snippets/fetch-user.md

```diff
@@ -0,0 +1,73 @@
+# On every URL change fetch users from the back end and deal with loading flags
```

Comment:
```suggestion
# Load data on route change

On every URL change fetch users from the back end and deal with loading flags
```

Reply:
an overall description aswell as a comparison (pros/cons) of the both given examples are missing